lib/grantinee/engine/postgresql.rb

```diff
@@ -37,9 +37,10 @@ def grant_permission!(data)
         table  = sanitize_table_name(data[:table])
         user   = sanitize_column_name(data[:user])
         fields = data[:fields].map { |v| sanitize_column_name(v.to_s) }.join(', ')
+        helper = kind == "EXECUTE" ? " FUNCTION " : ""
```

Comment:
We probably want to make this consistent with the `mysql` since it's the same behaviour... just have it here or in the conditional 🙂

Reply:
👍